e2e/content-services/document-list/document-list-pagination.e2e.ts

```diff
@@ -322,12 +301,91 @@ describe('Document List - Pagination', () => {
+        await browser.refresh();
+        await  paginationPage.clickItemsPerPageDropdown();
+        await expect(await paginationPage.getItemsPerPageDropdownOptions()).toEqual(['5', '10', '15', '20']);
+    });
+
+    it('[C272767] Should propagate the option chosen regarding displaying items per page to files/folders inside a folder', async () => {
+        await paginationPage.selectItemsPerPage(itemsPerPage.five);
+        await contentServicesPage.openFolder(newFolderModel.name);
+        await expect(await contentServicesPage.getActiveBreadcrumb()).toEqual(newFolderModel.name);
+        await expect(await paginationPage.getCurrentItemsPerPage()).toEqual(itemsPerPage.five);
+        await contentServicesPage.createAndOpenNewFolder(folderTwoModel.name);
+
+        for (let i = 0; i < numberOfSubFolders; i++) {
+            await contentServicesPage.createNewFolder('subfolder' + (i + 1));
+        }
+
+        await expect(await paginationPage.getPaginationRange()).toEqual('Showing 1-' + itemsPerPage.fiveValue + ' of ' + numberOfSubFolders);
+
+        await paginationPage.clickOnNextPage();
+        await expect(await paginationPage.getPaginationRange()).toEqual('Showing 6-' + numberOfSubFolders + ' of ' + numberOfSubFolders);
+        await contentServicesPage.openFolder('subfolder6');
+        for (let i = 0; i < numberOfSubFolders; i++) {
+            await contentServicesPage.createNewFolder('subfolder' + (i + 1));
+        }
+        await expect(await paginationPage.getPaginationRange()).toEqual('Showing 1-' + itemsPerPage.fiveValue + ' of ' + numberOfSubFolders);
+        await expect(await paginationPage.getCurrentPage()).toEqual('Page 1');
+        await expect(await paginationPage.getTotalPages()).toEqual('of 2');
+
+        await contentServicesPage.deleteSubFolderUnderRoot(newFolderModel.name, folderTwoModel.name);
+    });
+
+    it('[C260064] Should download only the last selection when changing pages in Single mode', async () => {
+        await paginationPage.selectItemsPerPage(itemsPerPage.five);
+        await contentServicesPage.openFolder(newFolderModel.name);
+        await expect(await paginationPage.getCurrentItemsPerPage()).toEqual(itemsPerPage.five);
+        await contentServicesPage.createAndOpenNewFolder(folderTwoModel.name);
+        for (let i = 0; i < numberOfSubFolders; i++) {
+            await contentServicesPage.createNewFolder('subfolder' + (i + 1));
+        }
+        await expect(await paginationPage.getPaginationRange()).toEqual('Showing 1-' + itemsPerPage.fiveValue + ' of ' + numberOfSubFolders);
+
+        await contentServicesPage.chooseSelectionMode('Single');
+
+        await contentServicesPage.selectFolder('subfolder1');
+        await paginationPage.clickOnNextPage();
+        await expect(await paginationPage.getPaginationRange()).toEqual('Showing 6-' + numberOfSubFolders + ' of ' + numberOfSubFolders);
```

Comment:
you can use `Showing 6-${numberOfSubFolders} of ${numberOfSubFolders}` instead of using the + operator
Please update every string on this PR that is using the + operator

Reply:
done